src/uu/runcon/src/runcon.rs

```diff
@@ -65,39 +57,29 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
                 match r.kind {
                     clap::ErrorKind::HelpDisplayed | clap::ErrorKind::VersionDisplayed => {
                         println!("{}", r);
-                        return error_exit_status::SUCCESS;
+                        set_exit_code(error_exit_status::SUCCESS);
```

Comment:
0 is automatically set as the exit code when `Ok(())` is returned

Reply:
I suppose it is safe to assume that `error_exit_status::SUCCESS` will always be 0 :). I'll make that change.

https://docs.rs/libc/latest/libc/constant.EXIT_SUCCESS.html